app.js

```diff
@@ -12,14 +14,146 @@ function toggleMobileMenu() {
   closeMenu.classList.toggle('show');
 }
 
-// Add an event listener
-toggleMobileMenu.addEventListener('click, mobileMenuList');
-
 // Close function for mobile-menu
 function closeMobileMenu() {
   mobileMenuList.classList.remove('show');
   mobileMenuButton.classList.remove('hidden');
 }
+mobileMenuButton.addEventListener('click', toggleMobileMenu);
+closeMenu.addEventListener('click', closeMobileMenu);
+
+const portfolioItemsDetail = [
+  {
+    id: '0',
+    name: 'Tonic',
+    year: '2015',
+    text: 'Lorem  of type and scrambled it 1960s. Lorem Ipsum is simplydummy text of the printing and typesetting industry.',
+    title: 'CANOPY',
+    stack: 'Back End Dev',
+    workStack: ['HTML', 'CSS', 'JavaScript'],
+    images: 'images/natureDesktop.png',
+    source: '#',
+    live: '#',
+  },
+
+  {
+    id: '1',
+    name: 'Multi-Post Stories',
+    year: '2015',
+    text: 'Lorem Ipsum is simply dummy text of the printing and  Lorem Ipsum is simplydummy text of the printing and typesetting industry.',
+    title: 'FACEBOOK',
+    stack: 'Full Stack Dev',
+    workStack: ['HTML', 'Ruby on Rails', 'CSS', 'Javascript'],
+    images: 'images/storiesDesktop.png',
+    source: '#',
+    live: '#',
+  },
 
-// An event listener
-closeMobileMenu.addEventListener('click', closeMobileMenu);
+  {
+    id: '2',
+    name: 'Facebook 360',
+    year: '2015',
+    text: 'Lorem Ipsum is simply dummy text of and scrambled it 1960s. Lorem Ipsum is simplydummy text of the printing and typesetting industry.',
+    title: 'Facebook',
+    stack: 'Full Stack Dev',
+    workStack: ['HTML', 'Ruby on Rails', 'CSS', 'Javascript'],
+    images: 'images/Facebook360Desktop.png',
+    source: '#',
+    live: '#',
+  },
+
+  {
+    id: '3',
+    name: 'Uber Navigation',
+    year: '2015',
+    text: 'Lorem Ipsum is simply dummy text of a galley of type and scrambled it 1960s. Lorem Ipsum is simplydummy text of the printing and typesetting industry.',
+    title: 'Uber',
+    stack: 'Lead Developer',
+    workStack: ['HTML', 'Ruby on Rails', 'CSS', 'Javascript'],
+    images: 'images/UberDesktop.png',
+    source: '#',
+    live: '#',
+  },
+];
+if (portfolio) {
+  for (let i = 0; i < portfolioItemsDetail.length; i += 1) {
+    const div = document.createElement('div');
+    div.classList.add('card');
+    div.setAttribute('id', `${i}`);
+    div.innerHTML = `
+   <div class="card-left-mobile">
+   <img src="${portfolioItemsDetail[i].images}" alt="snap-card">
+   </div>
+   <div class="card-right">
+     <h2>${portfolioItemsDetail[i].name}</h2>
+     <ul class="cano">
+       <li>${portfolioItemsDetail[i].title}<i class="fa fa-circle"></i></li>
+       <li>${portfolioItemsDetail[i].stack}<i class="fa fa-circle"></i></li>
+       <li>2015</li>
+     </ul>
+     <p>${portfolioItemsDetail[i].text}</p>
+     <ul class="lang">
+       ${portfolioItemsDetail[i].workStack
+    .map((stack) => `<li>${stack}</li>`)
+    .join('')}
```

Comment:
- [ ] You did a great job 👏 . However, you intriduced a new bug, and still need to fix the chnages suggested by the last reviewer 👍 
- Firstly, your cancel button has been omitted.

### View:
![image](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/50990925/671e3ecf-054c-4a10-b0d4-1031e8cf22b2)

### Figma design:
![image](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/50990925/145b25a3-eb1a-42c2-a58c-009198accb83)

- [ ] Secondly, just as the last reveiver pointed out, kindly adjust your popup. The buttom part should be visible on the screen, and fixed just as it is on figma. You can make reference  the materils for this week's for that 👍 

### Figma:
![image](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/50990925/60385406-7d77-4883-9f36-995e57cb7535)

### Your design:
![image](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/50990925/0a882244-d392-407d-bd4c-410046921610)